test/python/test_quantumprogram.py

```diff
@@ -1132,16 +1132,16 @@ def test_simulator_online_size(self, qe_token, qe_url):
         """
         backend_name = 'ibmq_qasm_simulator'
         q_program = QuantumProgram()
-        qr = q_program.create_quantum_register("q", 31)
-        cr = q_program.create_classical_register("c", 31)
+        qr = q_program.create_quantum_register("q", 1)
```

Comment:
We are going to try merging #791 first then.

Reply:
#791 was solved.